src/mapgen/mapgen.h

```diff
@@ -167,6 +167,7 @@ struct MapgenParams {
 */
 class Mapgen {
 public:
+	// Seed used for noises (truncated from the map seed)
 	s32 seed = 0;
```

Comment:
Would it be better to rename the variables to be more descriptive?

Reply:
Some day, maybe. But not now.